src/Components/Forms/src/FieldIdentifier.cs

```diff
@@ -15,7 +16,7 @@ namespace Microsoft.AspNetCore.Components.Forms;
 /// </summary>
 public readonly struct FieldIdentifier : IEquatable<FieldIdentifier>
 {
-    private static readonly ConcurrentDictionary<(Type ModelType, string FieldName), Func<object, object>> _fieldAccessors = new();
+    private static readonly ConcurrentDictionary<(Type ModelType, MemberInfo FieldName), Func<object, object>> _fieldAccessors = new();
```

Comment:
Nit: Should this be
```suggestion
    private static readonly ConcurrentDictionary<(Type ModelType, MemberInfo Member), Func<object, object>> _fieldAccessors = new();
```
?

Reply:
(Same suggestion for other occurrences in this PR)